Jenkinsfile

```diff
@@ -482,7 +482,7 @@ try {
 
                                     // windows docker container cannot reach instance-metadata endpoint. supply credentials at upload.
```

Comment:
This comment is somewhat misleading. The way we have Jenkins configured is that agents have access to the AWS role that the controller runs as by default, i.e. without the need to call `withAWS` at all.

It is good to explicitly assume the `ide-build` role using `withAWS`, as we are encouraging teams to use team specific roles in order to not overload the controller's role (we had previously maxed out the size of its policy).

Reply:
Leaving the `withAWS` block to be explicitly clear about what role we're using for the benefit of the next developer that reads it but isn't familiar with the default assumed AWS role